types/src/front/lib/core_api.ts

```diff
@@ -128,6 +126,13 @@ export type CoreAPITable = {
   parents: string[];
 };
 
+export type CoreAPITable = CoreAPITablePublic & {
```

Comment:
I notice that we don't suffix with "Type" here (contrary to other types I have seen so far), any reason why that I should be aware of ?

Reply:
I think we are suffixing with `Type` due to name-clashing with the models. `User` is the sequelize model, `UserType`  is the data-type that we typically return from `front`'s API

Reply:
I have the same rule in mind indeed 👍🏻